scrapy/pipelines/images.py

```diff
@@ -100,14 +101,20 @@ def file_downloaded(self, response, request, info):
     def image_downloaded(self, response, request, info):
         checksum = None
         for path, image, buf in self.get_images(response, request, info):
+            content_type = Image.MIME[image.format]
+            if self.convert_type != 'PRESERVE':
```

Comment:
This is an example of something to be tested (and documented).
I would say people would use `IMAGES_CONVERT_TYPE = None` (this is what the docs would recommend).
If you want to support blank values, it could also work for `if not self.convert_type`, but I would not document using blank values (you can add a test for this case though)